Flow.Launcher/Languages/sk.xaml

```diff
@@ -18,7 +16,7 @@
 
     <!--  Setting General  -->
     <system:String x:Key="flowlauncher_settings">Nastavenia Flow Launchera</system:String>
-    <system:String x:Key="general">Všeobecné</system:String>
+    <system:String x:Key="general">Všeobecn</system:String>
```

Comment:
Why is letter "é" cut off?

Reply:
I think some translation is loaded in a shared memory way? So a bit character seems being cut off? I don't know how crowdin implement that but I do find that chinese translation also get a bit cut off as well.

Reply:
Probably some special character are ignored there....Would you please do a proofread for that?

Reply:
All occurrences fixed!
I found that it's occurred if character "é" is followed by "<" only.

Reply:
Cool! I will sync it to crowdin